### PR TITLE
Create locale.sk.properties

### PR DESCRIPTION
**Pull request by misa1515**

### CODE REVIEW

File: files/files/translations/locale.sk.properties [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+# Shortcurs: hlprtx
+As HTML=jako &HTML
+As RTF=jako &RTF
+As TeX=jako &TeX
+As LaTeX=jako &LaTeX
+As PDF=jako &PDF
+As XML=jako &XML
+
+# File | Encoding menu
+# Shortcurs: 8bcklmu
```
Comment (moltenform) [inline files/files/translations/locale.sk.properties:58]: # Shortcuts: 8bcklmu

File: files/files/translations/locale.sk.properties [hunk 2]
```diff
@@ -0,0 +1,342 @@
...
+Export=&Exportovať
+Encoding=Kó&dovanie
+Page Setup=Vzhľad &stránky
+Print=&Tlač
+Load Session=Otvoriť &rozloženie
+Save Session=U&ložiť rozloženie
+Exit=&Koniec
+
+# File | Export menu
+# Shortcurs: hlprtx
```
Comment (moltenform) [inline files/files/translations/locale.sk.properties:49]: # Shortcuts: hlprtx

File: files/files/translations/locale.sk.properties [hunk 3]
```diff
@@ -0,0 +1,342 @@
...
+Run Current Macro=Spustiť súč&asné makro
+Record Macro=&Nahrať makro
+Stop Recording Macro=Zastaviť &nahrávanie makra
+Next Message=&Ďalšia správa
+Previous Message=Pr&edchádzajúca správa
+Clear Output=&Vyčistiť výstup
+Switch Pane=&Prepnúť panel
+
+# Options menu
+# Shortcuts: all except fghqwxy
```
Comment (moltenform) [inline files/files/translations/locale.sk.properties:150]: Was this meant to be left here?

**Comment by moltenform** [comment]
Thank you! I had a couple questions but I can merge it soon.
Many thanks for contributing to the project.

**Comment by misa1515** [comment]
Thank you. I used cz version like base, with mistakes.
Now used base from german/de.

**Comment by moltenform** [comment]
Done and added to the list!

https://github.com/moltenform/scite-files/blob/main/files/translations_list.md

**Pull request merged**